android/README.md

```diff
@@ -0,0 +1,21 @@
+# Ferrostar Android
+
+This directory tree contains the Gradle workspace for Ferrostar on Android.
+
+* `composeui` - Jetpack Compose UI elements which are not tightly coupled to any particular map renderer.
+* `core` - The core module is where all the "business logic", location management, and other core functionality lives.
+* `demo-app` - A minimal demonstration app.
+* `google-play-services` - Optional functionality that depends on Google Play Services (like the a fused location client wrapper). This is a separate module so that apps are able to "de-Google" if necessary.
+* `maplibreui` - Map-related user interface components built with MapLibre.
+
+## Running the demo app
+
+To run the demo app, you'll need a Stadia Maps API key
```

Comment:
Maybe we should create a fallback demo app ticket for iOS & Android. Just so devs can use the simulated route w/ demo tiles without having to sign up to stadia and get a key. Obviously this would be reduced demo functionality. While it's not a huge hurdle, it would keep the minimum requirements to contribute a little lower.

Reply:
Agreed. Captured it #270.